app/models/payroll/claim_csv_row.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require "delegate"
+require "csv"
+require "excel_utils"
+
+module Payroll
+  class ClaimCsvRow < SimpleDelegator
+    DATE_FORMAT = "%m/%d/%Y"
+
+    VALUES = [
+      UNITED_KINGDOM = "United Kingdom",
+      BASIC_RATE_TAX_CODE = "BR",
+      CUMULATIVE_TAX_BASIS = "0",
+      NOT_EMPLOYEES_ONLY_JOB = "3",
+      ALL_EMPLOYEES_NI_CATEGORY = "A",
+      HAS_STUDENT_LOAN = "T",
+      SCHEME_B_NAME = "Scheme B",
```

Comment:
So this isn't something you will have been privy too, but rather than saying "Scheme B" and "Scheme A" I think we're free to just say the policy name so there isn't extra effort required to translate between the policy and a magic "scheme" letter. Can we just say `"Student loans"` for now?

Reply:
Actually, can we some how get this from the claim? I'm thinking that'll make it better encapsulated for when Maths & Physics gets added.